### DISCUSSION TITLE
How can i get the ID of the USER?

### DISCUSSION THREAD

**Discussion by matukaking** [Glench/ExtPay#126] [Ideas]
I want to a build auth token with the ID of the user but i cant access it.

**Comment by Glench** [comment]
Can I ask why you’re wanting that information?

**Comment by dimadk24** [comment]
I'm currently investigating the usage of extpay for a Chrome extension. For me, the use case of the auth token is to store some user data on my server (both for free and paid users). I haven't tried extpay so far, but from what I see, it asks to login only when the user initiates a payment. As I understand (please correct me if I'm wrong), I either need to have two logins in the extension: one to get an authentication token for my server and one for extensionpay (user will be asked to log in only when pay), or implement stripe payments myself without extensionpay.
